tests/arm64/corefx_linux_test_exclusions.txt

```diff
@@ -0,0 +1,13 @@
+System.Buffers.Tests                 # timeout. baseline and jit stress.
+System.Drawing.Common.Tests          # bad result data. baseline and jit stress.
+System.Collections.Immutable.Tests   # JitStress=2
+System.Collections.Tests             # JitStress=2
+System.Linq.Expressions.Tests        # JitStress=1
+System.Linq.Tests                    # JitStress=2
+System.Net.NameResolution.Functional.Tests # baseline
+System.Net.NameResolution.Pal.Tests  # baseline
+System.Numerics.Vectors.Tests        # JitStress=2
+System.Runtime.Serialization.Formatters.Tests # baseline + many stress variants
+System.Security.Cryptography.Algorithms.Tests # bad result. baseline and jit stress.
+System.Text.RegularExpressions.Tests # https://github.com/dotnet/coreclr/issues/17754 -- timeout -- JitMinOpts + Tiered only
+System.Threading.Tests               # bad result. baseline
```

Comment:
I am surprised by the number of tests in this list.  I wasn't running jitstress much on corefx tests, so maybe those correspond to issues I knew about.

I think the timeouts are due to an issue running these in checked/debug.  There is an issue open somewhere about this.

I am particularly surprised by the System.Threading.Tests.  I was spending a lot of time looking at these tests to try to fix the SDK thread issues I was seeing.  Either this is a regression or I missed a failure (unlikely)...

Reply:
After I get testing in the CI (with these disabled), someone will need to run these and investigate. https://github.com/dotnet/coreclr/issues/19987 tracks this.